### DISCUSSION TITLE
Unsupported JPEG data precision 16

### DISCUSSION THREAD

**Discussion by stemps** [ImageMagick/ImageMagick#6531] [Help] [answered]
### ImageMagick version

7.1.1.-15

### Operating system, version and so on

Mac OS 14.0 (beta)

### Description

I'm trying to convert a 16-bit lossless jpeg from an MRT scan in a DICOM file via [ruby-dicom](https://github.com/dicom/ruby-dicom) and [minimagick](https://github.com/minimagick/minimagick) and I'm getting the error:

```mogrify: Unsupported JPEG data precision 16```

ImageMagick is installed via homebrew on a Mac. Can I get ImageMagick to work with these formats?

**Comment by dlemstra** [comment, marked as answer]
You cannot get ImageMagick to work with this because 16-bit jpeg images are not supported. That is the reason that you are getting this error message.
